scripts/update-whitelist.ts

```diff
@@ -0,0 +1,72 @@
+// add and remove must be comma delimited strings
+//-----------------------------------------------
+
+import { SigningCosmWasmClient } from '@cosmjs/cosmwasm-stargate';
+import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
+import { calculateFee, coins, GasPrice } from '@cosmjs/stargate';
+
+const config = require('./config');
+
+async function updateWhitelist(add: string, remove: string) {
+  const add_addresses = add == '' ? null : add.split(',');
+  const remove_addresses = remove == '' ? null : remove.split(',');
+  if (add_addresses != null) {
+    console.log('add addresses: ', add_addresses.join(','));
+  }
+  if (remove_addresses != null) {
+    console.log('remove addresses: ', remove_addresses.join(','));
+  }
+
+  const gasPrice = GasPrice.fromString('0stars');
+  const wallet = await DirectSecp256k1HdWallet.fromMnemonic(
+    config.mnemonic,
+    {
+      prefix: 'stars',
+    },
+  );
+  const client = await SigningCosmWasmClient.connectWithSigner(
+    config.rpcEndpoint,
+    wallet,
+  );
+  const executeFee = calculateFee(300_000, gasPrice);
```

Comment:
Have you tried this gas with a very large whitelist?

Reply:
I tried with a very large list, 500+ addresses, and it runs out of gas. We should probably have a file option.

Reply:
Actually is there an array or message size limit imposed by the blockchain? In terra you can not send arrays over size 50.